src/Client/Invoice.php

```diff
@@ -207,33 +207,6 @@ public function markInvoiceStatus(string $storeId, string $invoiceId, string $ma
         }
     }
 
-    public function updateInvoice(
```

Comment:
really strange that this is here even that I merged your other PR to 3.x already, wth?

Reply:
it's my fault, i am very stupid with git. After this is merged i'm going to delete everything from PC and just clone it fresh